proxy/src/graphql/schema.rs

```diff
@@ -149,6 +150,10 @@ impl Query {
         "1.0"
     }
 
+    fn avatar(handle: juniper::ID) -> Result<avatar::Avatar, error::Error> {
+        Ok(avatar::Avatar::from(&handle.to_string()))
+    }
```

Comment:
Shouldn't we return `Err` if there is an error?

Reply:
None of the operations here a fallible so far, so there will never be an error at the moment. This might change down the road.